### DISCUSSION TITLE
AWS CloudWatch Logs

### DISCUSSION THREAD

**Discussion by timoweiss** [vercel/community#733] [Idea]
Hi there,

we would love to store the logs produced by our application on AWS CloudWatch Logs.
We tried to send them from within the process but only with limited success.
Once the traffic increases, we hit a client connection limit (from CloudWatch). If we batch the logs, batches regularly get lost as the lambda is shut down before they are flushed.

Are there any plans to support an AWS CloudWatch adapter for log drains in the near future?

Thank you very much in advance,

Timo

**Comment by uncledru** [comment]
Hello @timoweiss 👋🏽 

I'm also looking into creating an integration for CloudWatch Logs. I'm curious as to what you have tried so far? Did you create a custom integration yourself?

**Comment by dossy** [comment]
I would also like a built-in log drain target for AWS CloudWatch Logs.  In the meantime, it looks like the best option might be to set up a Lambda that acts as an intermediary between Vercel's generic Log Drain and CWL.

https://dev.to/mannieschumpert/setting-up-a-vercel-log-drain-with-aws-4a9c

**Comment by backstromjoel** [comment]
Any updates here? Would be very nice to see a more robust integration than the lambda one mentioned above.
